### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by aletail**
https://werkbotstudios.teamwork.com/#/tasks/27333196

I consider this the first pass on the documentation, some things I was not sure of as far as setup.

Just read through and suggest edits or anything we missing

### CODE REVIEW

File: docs/en/sass-setup.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
+# SASS Configuration
+
+The sass is built to be configurable per project, we do this by having a default properties map which we refer to as the "theme", its located here: `sass\variables\_theme.scss`.
```
Comment (saraolives) [inline docs/en/sass-setup.md:3]: you can probably take out "its" or change it to "it's"

File: docs/en/sass-setup.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  tertiaryColor: #cccccc,
+  backgroundColorLight: #ffffff,
+  backgroundColorDark: #333333,
+  textColorLight: #ffffff,
+  textColorDark: #333333,
+
+  // Fonts
+  fontHeader: "Montserrat",
+  fontText: "Roboto",
+  fontIcon: "Font Awesome 5 Pro",
```
Comment (saraolives) [inline docs/en/sass-setup.md:19]: Should we use the free library here as an example?

**Review by saraolives — CHANGES_REQUESTED** [review]
This looks great! I just found a few little typos mainly

File: docs/en/sass-setup.md (hunk 2 above)
Comment (aletail) [reply docs/en/sass-setup.md:19]: Yes, thats a good idea!

**Review by saraolives — APPROVED** [review]
Looks great!

**Pull request merged**